PWGHF/vertexingHF/AliAnalysisTaskSED0BDT.h

```diff
@@ -435,10 +436,12 @@ class AliAnalysisTaskSED0BDT : public AliAnalysisTaskSE
   Bool_t    fEnablePileupRejVZEROTPCcls;
   Bool_t    fRejectOutOfBunchPileUp;
   
-  TH3F *h3Invmass[5];     //!<!
-    TH3F *h3Invmass_19[5];     //!<!
-    TH3F *h3Invmass_1029[5];     //!<!
-    TH3F *h3Invmass_3059[5];     //!<!
+  TH3F *h3Invmass[8];     //!<!
+    TH3F *h3Invmass_19[8];     //!<!
+    TH3F *h3Invmass_1029[8];     //!<!
+    TH3F *h3Invmass_3059[8];     //!<!
+    TH3F *h3Invmass_19999[8];     //!<!
+    TH3F *h3Invmass_6099[8];     //!<!
```

Comment:
Hi @xinyepeng, since you modified and added some data members you should increase by 1 the ClassDef, once you will do it I will give you +1. Thanks!

Reply:
ciao @fgrosa, thanks for pointing it out, it's done, thanks to you! And happy holiday!

Reply:
Thanks! And happy holidays to you too in case you will have some day of vacation!